### DISCUSSION TITLE
Single page view

### DISCUSSION THREAD

**Discussion by Pollomonteros** [johnfactotum/foliate#1166] [General]
Is there a way to view a single page of a book instead of two side to side ? I am using a language textbook while taking notes with Obsidian and when having both apps open side to side Foliate pages look too small. 
I managed to find an older thread from some years ago but it was closed without an answer so sorry if I am missing something

**Comment by johnfactotum** [comment]
From the book viewer, press <kbd>Alt</kbd> + <kbd>,</kbd>, or click on the View menu and choose "Font & Layout Settings".

Then in the "Layout" page, scroll down to the "Page" section. Here you can increase "Maximum Inline Size", or set "Maximum Number of Columns" to 1. 

Alternatively, you can try using the scrolled mode (<kbd>Ctrl</kbd> + <kbd>M</kbd>).
